channel-utilization/channel_utilization_sqlite.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Define the output CSV file
+output_file="temp.csv"
+
+# Create CSV header
+echo "timestamp_ns,chan_id_in,chan_id_out,amt_in_msat,amt_out_msat,fee_msat" > "$output_file"
+
+# Initialize counter
+counter=0
+batch_size=10000
+
+# Function to process a batch of data
+process_batch() {
+    local offset=$1
+    sqlite3 -csv -noheader circuitbreaker.db <<EOF
+SELECT add_time, incoming_channel, outgoing_channel, incoming_amt_msat, outgoing_amt_msat,
```

Comment:
Would you mind updating this to add `resolved_time` - just added this in the other script in #3 because I realized we're also going to need that value!